packages/flutter_tools/lib/src/macos/xcode.dart

```diff
@@ -117,9 +136,9 @@ class Xcode {
     );
   }
 
-  Future<String> iPhoneSdkLocation() async {
+  Future<String> sdkLocation(SdkType sdk) async {
```

Comment:
If you add new public API surface, these need tests. (Feel free to do them in a separate PR)

Reply:
iPhoneSdkLocation didn't have any tests to update.  Will add some.

Reply:
https://github.com/flutter/flutter/issues/46137